src/components/HomePage.js

```diff
@@ -135,49 +116,27 @@ class HomePage extends Component {
                   </p>
 
                   <p>
-                    I am MUGIRASE Emmanuel from Gisenyi, Rwanda. I have worke in health sector
-                    as an assistant clinical anaesthetist since 2014 till 2020, where I was an
-                    assistant of anesthesiology head of department in Gisenyi Hospital, I have
-                    worked as team coordinator, and anesthesia services providing for the entire
-                    time I spent there. I worked as an english teacher in Muduha Primary school
-                    in 2011. After all of those works I shifted my career to software engineering
-                    which I dreamt for years, I started software engineering in 2017, where I
-                    learned Java, Javascript(NodeJS, React), Android, Web technologies.
-                  </p>
-
-                  <p>
-                    I am confident enough to work on any health system app (desktop app,
-                    web app, entreprise app, or mobile app), I am an innovative man,
-                    who loves working with teams on daily basis.
-                  </p>
-
-                  <p>
-                    I graduated from University of Rwanda in College of Medicine
-                    and Health sciences in 2014, and in 2016 I went to Kigali independent
-                    University to study computer sciences, when I grabbed the basics of programming
-                    I dropped out before graduating because I wanted to focus muore on software
-                    engineering in a professiona way. Now I am ready to bring solutions using
-                    programming.
+                   I am a full-stack software engineer with expertise in NodeJS, PostgreSQL, React, Ruby, Ruby on Rails, 
+                   Semantic HTML5, CSS3, Git, Agile methodology, Mobile applications, Technical Team Lead, and mentor.
```

Comment:
Codacy found an issue: [This line has a length of 118. Maximum allowed is 100.](https://app.codacy.com/manual/descholar-ceo/portfolio/pullRequest?prid=6035379)